src/pages/resources/reference/push-notifications/templates/rating.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Adobe Experience Platform SDK timer push template
+description: Timer push template supported by Adobe Campaign Classic mobile extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Rating
+---
+
+import Tabs from './tabs/rating.md'
+
+# Push template - rating
+
+A notification that allows displaying a title, text, and image, along with 3-5 image buttons that allow the user to provide "rating" feedback.
+
+After a rating is selected in the notification, an intent containing the rating is dispatched to the app.
```

Comment:
should we capitalize and monospace `Intent`, given it's a class name?

Reply:
yes i think this is a good idea. i'll update here and throughout.